docs/how-to/ccv0.sh

```diff
@@ -329,13 +329,11 @@ install_guest_kernel_image() {
 
 build_qemu() {
     ${tests_repo_dir}/.ci/install_qemu.sh
+    ${tests_repo_dir}/.ci/install_virtiofsd.sh
```

Comment:
This is a nit, but could you have both either before or after installing the VMM? Just for the sake of consistency.

Reply:
That's completely fair and just sloppy copy-pasta from me. Fixed now